pkg/cdi/container-edits.go

```diff
@@ -220,3 +286,72 @@ func ensureOCIHooks(spec *oci.Spec) {
 		spec.Hooks = &oci.Hooks{}
 	}
 }
+
+// fillMissingInfo fills in missing mandatory attributes from the host device.
+func fillMissingInfo(dev *oci.LinuxDevice) error {
+	if dev.Type != "" && (dev.Major != 0 || dev.Type == "p") {
+		return nil
+	}
+	hostDev, err := runc.DeviceFromPath(dev.Path, "rwm")
```

Comment:
Does this mean that we require `rwm` permissions on the host device?

Reply:
No, that's just blindly copied into the returned structure. (IIUC) `rwm` happens to be the fallback default elsewhere, so I went with that. Shouldn't affect the outcome.